src/commands/InspectCommand.ts

```diff
@@ -0,0 +1,77 @@
+import {Message, MessageEmbed} from "discord.js";
+import Command from "../abstracts/Command";
+import DateUtils from "../utils/DateUtils";
+import {EMBED_COLOURS} from "../config.json";
+
+class InspectCommand extends Command {
+	constructor() {
+		super(
+			"inspect",
+			"Show information about a given user"
+		);
+	}
+
+	async run(message: Message, args: string[]) {
+		const embed = new MessageEmbed();
+
+		try {
+			let userObj;
+			let userList;
+
+			if (args.length > 0) {
+				if ((/^[0-9]+$/g).test(args[0])) {
+					// - args[0] only contains numbers so its a user ID
+					userObj = await message.guild?.members?.fetch(args[0]);
+				} else {
+					// If args[0] does not match username#0000 throw error
+					if (!(/^.*#[0-9]{4}$/).test(args[0])) throw "";
+
+					const userData = args[0].split("#");
+
+					const username = userData[0];
+					const discriminator = userData[1];
```

Comment:
Can be simplified to:
```suggestion
					const [username, discriminator] = args[0].split("#");
```